include/fiber_utils.h

```diff
@@ -16,6 +16,20 @@ typedef struct{
     "Example of correct syntax: test3D_C2C -lib heffte   -backend fftw    -size nx ny nz      -pgrid P Q    -iter Niter \n"
     "                                      -comm a2a/a2av/a2aw/p2p    -reshape pencil/slabs/bricks   -trace   \n";
 
+enum {
+ backend_option_fft_op = 0, // 0
+ backend_option_nx,         // 1
+ backend_option_ny,         // 2
+ backend_option_nz,         // 3
+ backend_option_backend,    // 4
+ backend_option_grid_p,     // 5
+ backend_option_grid_q,     // 6
+ backend_option_physical,   // 7
```

Comment:
Instead of using backed_option_<....>. it looks better just to start with option_<.....>
For example backends_options[backend_option_backend], uses many "backend" words.

Reply:
I updated the enum as directed:

- change the value of the option_physical from 7 to 13
- remove 'backend_' for each option